Source/WebCore/platform/graphics/FontDescription.h

```diff
@@ -31,12 +31,25 @@
 #include "TextFlags.h"
 #include "WebKitFontFamilyNames.h"
 #include <unicode/uscript.h>
+#include <wtf/Markable.h>
 #include <wtf/MathExtras.h>
 
 namespace WebCore {
 
 using namespace WebKitFontFamilyNames;
 
+struct FloatMarkableTraits {
```

Comment:
Should we share this in `Markable.h` ?

`Markable<float>` will be useful in plenty of situation on the codebase.

Reply:
I considered that, the only thing that me question it is because we can't pass the empty value through a template argument, since it's disallowed to pass floats as arguments, so it makes the traits a little less flexible. Maybe I can still move it there and somehow make it explicit from the name that it'll use NAN as the empty value